northstar/src/runtime/state.rs

```diff
@@ -457,10 +459,13 @@ impl State {
                 Error::Io(format!("Failed to read {}", repository.dir.display()), e)
             })?;
             while let Ok(Some(entry)) = dir.next_entry().await {
-                let manifest = ArchiveReader::new(&entry.path().as_path(), None)
-                    .map_err(Error::Npk)?
-                    .manifest()
-                    .clone();
+                let manifest = tokio::task::block_in_place(|| {
+                    let file = File::open(entry.path().as_path()).map_err(|e| {
+                        npk::npk::Error::FileOperation(format!("Failed to open NPK: {}", e))
```

Comment:
` npk::npk::Error::FileOperation` is a `Io` error. In all the `Error` enums we have a `Io` that wraps a `io::Error`. Think we should rename this to `Io`.